composer.json

```diff
@@ -23,18 +23,16 @@
   ],
   "homepage": "https://github.com/bepsvpt/secure-headers",
   "require": {
-    "php": "^7.0 || ^8.0",
-    "illuminate/support": "~5.1 || ~6.0 || ~7.0 || ~8.0 || ~9.0"
+    "php": "^7.0 || ^8.0"
   },
   "require-dev": {
     "ext-json": "*",
     "ext-xdebug": "*",
-    "orchestra/testbench": "~3.1 || ~4.0 || ~5.0 || ~6.0 || ~7.0"
-  },
-  "suggest": {
```

Comment:
Consider retaining a `suggest` block with `illuminate/support` since you still have SecureHeadersMiddleware and SecureHeadersServiceProvider which depends on that package. Consumers of this package may not install the devDependencies, and it would be helpful for Laravel users to have a notice to install that package if they will be using SecureHeadersMiddleware or SecureHeadersServiceProvider.

Reply:
It is contradictory that put the `illuminate/support` in the `suggest` section. When the user installs this package on a Laravel project, the suggestion will not be shown; and when the user installs this package on a non-Laravel project, this suggestion is useless for that user. This is the reason that I just removed it from the dependencies.

Reply:
Okay, yeah that makes sense. And I imagine it's a common package that Laravel projects will probably already have installed.